apps/wappush/js/wappush.js

```diff
@@ -193,18 +193,17 @@
 
     var title = Utils.prepareMessageTitle(message);
 
-    var options = {
+    NotificationHelper.send(title, {
```

Comment:
Same thing here. Utils.prepareMessageTitle returns a result of mozL10n.get.

You either have to wrap it and pass {raw: title}, or you can refactor prepareMessageTitle to return {id:, args} and then you can pass it here.

If you want to take the refactor, I'd prefer that. Then, you also need to update the other use of prepareMessageTitle in apps/wappush//js/cp_screen_helper.js to do navigator.mozL10n.setAttributes(node, l10n.id, l10n.args) on those three nodes there.